client/driver/rkt.go

```diff
@@ -53,20 +55,27 @@ type RktDriverConfig struct {
 
 // rktHandle is returned from Start/Open as a handle to the PID
 type rktHandle struct {
-	proc        *os.Process
-	image       string
-	logger      *log.Logger
-	killTimeout time.Duration
-	waitCh      chan *cstructs.WaitResult
-	doneCh      chan struct{}
+	pluginClient *plugin.Client
+	userPid      int
+	executor     executor.Executor
+	taskDir      string
+	allocDir     *allocdir.AllocDir
```

Comment:
Why do we need the allocDir in the handle and the rktPid? I don't see allocDir being used in Open at all.